PublicationsRouter/api/v3/Send.md

```diff
@@ -234,41 +232,42 @@ Note these are different from the Validation endpoint.
     {
         "successful": <number of successfully processed notifications>,
         "total": <the number of items received in the list>,
-        "success_ids": [ <list of IDs of successfully processed notifications> ],
-        "fail_ids": [ <list of IDs of notifications that could not be processed> ],
-        "last_error": "A notification in the list is not a JSON object, the id of the latest
-                    notification processed was '5'. Error: <human readable error message>"
+        "created_ids": [ <list of PubRouter notification IDs of created notifications> ],
+        "success_ids": [ <list of submitted IDs of successfully processed notifications> ],
+        "fail_ids": [ <list of submitted IDs of notifications that could not be processed> ],
+        "last_error": <error message describing the error which caused the last failed notification to fail>
     }
 ```
 
-#### Success Response - Single Notification ####
-* On **successful completion** of the request, the system will respond with 201 (Accepted) and the following response body
+
+#### Success Response - Notification List ####
+* On **successful completion** of the request, the system will respond with 201 (Created) and the following response body.  
 
 ```
     HTTP 1.1  201 Created
     Content-Type: application/json
-    Location: <url for api endpoint for accepted notification>
 
     {
-        "status" : "accepted",
-        "id" : "<unique identifier for the notification>",
-        "location" : "<url path for api endpoint for newly created notification>"
+        "successful": <number of successfully processed notifications>,
+        "total": <the number of items received in the list>,
+        "success_ids": [ <list of IDs of successfully processed notifications> ],
+        "fail_ids": [ <list of IDs of notifications that could not be processed> ],
+        "last_error": "<Last error message>"
     }
 ```
 
-#### Success Response - Notification List ####
-* On **successful completion** of the request, the system will respond with 202 (Accepted) and the following response body.  Note you may obtain this successful notification even if some of the notifications in the list could not be processed - these are identified in the fail_ids list.
+#### Success Response - Single Notification ####
+* On **successful completion** of the request, the system will respond with 201 (Accepted) and the following response body
 
 ```
     HTTP 1.1  201 Created
     Content-Type: application/json
+    Location: <url for api endpoint for accepted notification>
 
     {
-        "successful": <number of successfully processed notifications>,
-        "total": <the number of items received in the list>,
-        "success_ids": [ <list of IDs of successfully processed notifications> ],
-        "fail_ids": [ <list of IDs of notifications that could not be processed> ],
-        "last_error": "<Last error message>"
+        "status" : "accepted",
```

Comment:
Should this have `status` now?

Reply:
confusingly that is how the `create_notification` endpoint works (for single notifications) I'm quite happy to change the code so that it no longer includes status in the json response

Reply:
I've changed the code to accommodate this